src/sst/core/baseComponent.cc

```diff
@@ -168,6 +168,14 @@ BaseComponent::registerClock(const UnitAlgebra& freq, Clock::HandlerBase* handle
     return tc;
 }
 
+TimeConverter*
+BaseComponent::registerClock(TimeConverter& tc, Clock::HandlerBase* handler, bool regAll)
```

Comment:
I'm not entirely sure we need to pass by reference, but if we do, we should probably pass by const reference.